client/util/time.ts

```diff
@@ -31,3 +32,19 @@ export function secondsToTime(s: number): string {
 
     return format("year")
 }
+
+export function secondsToTimeExact(s: number): string {
```

Comment:
Why not just use `secondsToTime`, or change `secondsToTime`?

Reply:
`secondsToTime` is used for relative post time and in some languages it has a different grammatical form than it's needed when denoting video length in playlist thus language neutral representation.

Reply:
That makes sense.